samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -119,13 +119,25 @@ var AppController = function(loadingParams) {
     this.call_.oncallerstarted = this.displaySharingInfo_.bind(this);
 
     this.roomSelection_ = null;
-    // If the params has a roomId specified, we should connect to that room immediately.
-    // If not, show the room selection UI.
+    this.localStream_ = null;
+
+    // If the params has a roomId specified, we should connect to that room
+    // immediately. If not, show the room selection UI.
```

Comment:
This function is getting quite long. Consider breaking out this UI setup code into its own function.